activiti-cloud-connectors-starter/src/test/java/org/activiti/cloud/connectors/starter/test/it/ActivitiCloudConnectorServiceIT.java

```diff
@@ -43,17 +45,21 @@
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
-@EnableBinding(RuntimeMockStreams.class)
+@EnableBinding({RuntimeMockStreams.class, MockProcessEngineChannels.class})
```

Comment:
The `@EnableBinding` annotation should not be attached directly to the test class, otherwise the related stream listeners will be loaded even [when other tests are running](https://github.com/Activiti/Activiti/issues/1501). This annotation should be moved to a dedicated component that is loaded only when a [specific profile is enabled](https://github.com/Activiti/activiti-cloud-runtime-bundle-service/pull/3/files#diff-455a95e95c031b63b46367002a20ee86R19).